CMakeLists.txt

```diff
@@ -55,42 +55,42 @@ set(GEOSChem_DETECTED_FORTRAN_COMPILER_VERSION ${CMAKE_Fortran_COMPILER_VERSION}
 )
 
 set(GEOSChem_Fortran_FLAGS_Intel
-    -cpp -w -auto -noalign "SHELL:-convert big_endian" "SHELL:-fp-model source"
+    -g -cpp -w -auto -noalign "SHELL:-convert big_endian" "SHELL:-fp-model source"
     -mcmodel=medium -shared-intel -traceback -DLINUX_IFORT
     CACHE STRING "GEOSChem compiler flags for all build types with Intel compilers"
 )
 set(GEOSChem_Fortran_FLAGS_RELEASE_Intel
    -O2
-    CACHE STRING "GEOSChem compiler flags for build type release with Intel compilers"
+    CACHE STRING "GEOSChem compiler flags for build type Release with Intel compilers"
 )
 set(GEOSChem_Fortran_FLAGS_RELWITHDEBINFO_Intel
     -O2
-    CACHE STRING "GEOSChem compiler flags for build type relwithdebinfo with Intel compilers"
+    CACHE STRING "GEOSChem compiler flags for build type RelWithdDebInfo with Intel compilers"
 )
 set(GEOSChem_Fortran_FLAGS_DEBUG_Intel
-    -g -O0 "SHELL:-check arg_temp_created" "SHELL:-debug all" -fpe0 -ftrapuv -check,bounds
-    CACHE STRING "GEOSChem compiler flags for build type debug with Intel compilers"
+    -O0 "SHELL:-check arg_temp_created" "SHELL:-debug all" -fpe0 -ftrapuv -check,bounds
```

Comment:
Why is debugging information now removed when building in DEBUG mode? Same comment for GNU.